css/style.css

```diff
@@ -1,211 +1,448 @@
-body {
+/* ----------Fonts---------- */
+
+.title,
+.h1Top,
+.h1Bottom,
+.linkHeader,
+.sendButton,
+.sectionButton,
+.subscribeButton {
 	font-family: Georgia, 'Times New Roman', Times, serif;
 }
+
+.linkDescription {
+	font-family: Tahoma, 'Segoe UI', Geneva, Verdana, sans-serif;
+}
+
+footer,
+.underImg,
+.section p,
+.buttonDiv,
+.underTitle,
+.bottomRightIcons p {
+	font-family: Helvetica, Arial, sans-serif;
+}
+
+.h1BottomBottom {
+	font-family: 'Gill Sans MT', 'Gill Sans', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.namePerson,
+.messagePerson,
+.emailPersonLeft,
+.emailPersonCenter {
+	font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+
+/* ----------Others---------- */
+
+
+.hr2 {
+	margin-top: 0;
+	margin-bottom: 0;
+}
```

Comment:
this is the same as margin: 0 auto. Use these when you know that there is margin left or margin right that are given to this element other way.